.bazelrc

```diff
@@ -30,3 +30,6 @@ build --action_env=OVA_PROVIDER_SERVER_IMAGE=quay.io/kubev2v/forklift-ova-provid
 # sandboxes. Use slightly less secure but working processwrapper sandbox.
 # NOTE: Same configuration is in virt-v2v/cold/.bazelrc.
 build --strategy_regexp="Action appliance/libguestfs-appliance.tar"=processwrapper-sandbox
+
+# For populator images, we need to use processwrapper sandbox as well
+build --strategy_regexp="Action ovirt-populator-image"=processwrapper-sandbox
```

Comment:
@ahadas do we want to just always use it?

Reply:
I think it's good to stick with the default when it works